runtime/src/test/java/com/redhat/ipaas/runtime/ComponentsITCase.java

```diff
@@ -26,15 +26,15 @@
     @Test
     public void componentsListTest() {
         ListResult<Component> results = restTemplate().getForObject("/api/v1/components", ListResult.class);
-        assertThat(results.getItems()).hasSize(20);
-        assertThat(results.getTotalCount()).isEqualTo(50);
+        assertThat(results.getItems()).hasSize(2);
+        assertThat(results.getTotalCount()).isEqualTo(2);
     }
 
     @Test
     public void componentsGetTest() {
-        Component result = restTemplate().getForObject("/api/v1/components/1", Component.class);
+        Component result = restTemplate().getForObject("/api/v1/components/org.foo:twitter-mention-connector:1.0", Component.class);
```

Comment:
I know it's a breaking change to the API, but shouldn't this be `connectors` rather than `components` now? Same change for the kind in the deployment.json too I would recommend.